qutebrowser/browser/webelem.py

```diff
@@ -50,6 +50,9 @@
     Group.url: '[src], [href]',
     Group.prevnext: 'a, area, button, link, [role=button]',
     Group.focus: '*:focus',
+    Group.inputs: ('input[type=text], input[type=email], input[type=url], '
+                   'input[type=tel], input[type=number], input[type=password], '
```

Comment:
nitpick, please break `input[type=password]` onto the next line as it's too long otherwise